### DISCUSSION TITLE
Decouple daily backups and automatic updates

### DISCUSSION THREAD

**Discussion by lars-olsen** [nextcloud/all-in-one#3249] [Questions] [answered]
Currently it's only possible to have daily backups and automatic updates enabled together. I like daily backups, but not necessarily automatic updates of all containers and apps. 

It does of course make sense to check that there is a recent backup before updating anything, so maybe check if auto backup is enabled, when the user enables automatic container and app updates?

**Comment by szaimen** [comment, marked as answer]
Hi, this is already possible or am I misunderstanding something?
![image](https://github.com/nextcloud/all-in-one/assets/42591237/a3a76452-ad45-4e93-8c66-185c991dc74a)

**Comment by lars-olsen** [comment]
Pretty sure i did not get that possibility when i enabled automatic backups for the first time. It's there now, which is much appreciated.
